sources/getUpdatedWorkspaces/getUpdatedWorkspaces.test.ts

```diff
@@ -1,8 +1,8 @@
 import { initializeTestRepository } from '../../testing/repository';
 import getUpdatedWorkspaces from './getUpdatedWorkspaces';
 
-describe('getUpdatedWorkspaces는', () => {
-  it('Workspace 내부의 업데이트된 패키지 리스트를 반환한다.', async () => {
+describe('getUpdatedWorkspaces', () => {
```

Comment:
(사소) 영어라면 `describe('Something', () => { it('does some other thing', ...) })` 형식이 자연스럽지만 한국어면 `X는 Y한다.` 형식이 더 자연스럽게 테스트 결과가 나올 거라고 생각했었는데 혹시 어떻게 생각하시나요? 

<img width="684" alt="image" src="https://user-images.githubusercontent.com/3102175/134455557-500b24a0-65b1-488c-b371-edd4a64f32ce.png">

예를 들어서 현재 상태는
```
getUpdatedWorkspaces
  ㄴ Workspace 내부의 업데이트된 패키지 리스트를 반환한다
```

이런 식으로 보이게 될 것 같아서 예전보다 자연스러움이 떨어지게 될 거라고 생각했었어요

Reply:
나머지 테스트와 톤을 통일하려고 한 것이긴 한데, 저는 조사가 없는 구조도 큰 문제는 없다고 생각합니다. 말씀하신 케이스에서는 조사가 있는 텍스트가 더 자연스러운데, ListCommand 테스트 같은 경우에는 조사를 적용하기 힘들 것 같아서 없는 쪽으로 통일했습니다.
```
- ListCommand
  - yarn workspaces since list <from>
    - 변경된 package가 있다면 해당 package들을 출력한다
  - yarn workspaces since list <from> <to>
    - <to>까지 변경된 package들을 출력한다
```